### ISSUE TITLE
Tree is gone

### ISSUE THREAD

**Issue by Daniboay** [vaadin/framework8-issues#543]
In Vaadin 8 alpha 8 the Tree component seems to be missing.
I was reading about a HierarchicalDataSource in this issue: https://github.com/vaadin/framework8-issues/issues/82
But that seems not to be available in the alpha build.
Any suggestion how to replace the Vaadin 7 tree component in Vaadin 8?

**Comment by pleku** [comment]
Tree and hierarchical data provider support has been postponed to 8.1.

For 8.0, you can keep using the Framework 7 Tree which is located in the compatibility packages, which you can get by adding the following dependencies 
```xml
        <dependency>
            <groupId>${project.groupId}</groupId>
            <artifactId>vaadin-compatibility-client-compiled</artifactId>
            <version>${project.version}</version>
        </dependency>
        <dependency>
            <groupId>${project.groupId}</groupId>
            <artifactId>vaadin-compatibility-server</artifactId>
            <version>${project.version}</version>
        </dependency>
```

further details about migrating will be available later (#187)

**Comment by pleku** [comment]
Duplicate of #306, #318
